packages/requester-utils/src/RequesterUtils.ts

```diff
@@ -53,16 +54,28 @@ export function defaultOptionsHandler(
   resourceOptions: DefaultResourceOptions,
   { body, query, sudo, method = 'get' }: DefaultRequestOptions = {},
 ): DefaultRequestReturn {
-  const { headers, requestTimeout, url } = resourceOptions;
+  const { headers, requestTimeout, url, additionalBody = {} } = resourceOptions;
   let bod: FormData | string;
 
   if (sudo) headers.sudo = sudo;
 
   // FIXME: Not the best comparison, but...it will have to do for now.
   if (typeof body === 'object' && body.constructor.name !== 'FormData') {
-    bod = JSON.stringify(decamelizeKeys(body));
+    bod = JSON.stringify(decamelizeKeys({ ...body, ...additionalBody }));
     headers['content-type'] = 'application/json';
   } else {
+    /**
+     * TODO - what do I do here with the additionalBody?
+     *
+     * Update: attempting to parse `body` like so:
```

Comment:
How is the additional information sent when their is a form data request? Should they be added to the formdata? If so this modification will probably have to happen in the specific requesters (Ky and Got) because they handle formdata differently

Reply:
hmm i probably never encountered this case or something because it was never an issue, but how to solve it - no clue..

Reply:
Did you ever try to import or export projects using the api? Thats really the only time the formdata is called on.

Reply:
nope, only used gitbeaker inside my own extension [1] but nowhere else..

[1] http://github.com/kiprasmel/refined-gitlab